src/test/ui/coherence/issue-99663.rs

```diff
@@ -0,0 +1,20 @@
+// check-pass
+
+#![feature(type_alias_impl_trait)]
+
+struct Send<T> {
+    i: InnerSend<T>,
+}
+
+type InnerSend<T> = impl Sized;
+
+fn constrain<T>() -> InnerSend<T> {
+    ()
+}
+
+trait SendMustNotImplDrop {}
+#[allow(clippy::drop_bounds, drop_bounds)]
```

Comment:
```suggestion
```

Reply:
I need `allow(drop_bounds)` because this is a `// check-pass` test, or else it'll fail due to the warn.